scripts/Calibration/tofpd/diagnostics.py

```diff
@@ -74,6 +84,36 @@ def _get_difc_ws(wksp, instr_ws=None):
     return difc_ws
 
 
+def __get_regions(x):
+    # Returns a list of tuples with start,stop indices
```

Comment:
can you include a formal docstring so that a description for input parameter `x` is also present?